Overflow Error - ZeroInflatedBinomial

**Describe the bug**
Currently the `test_auto_recover` for `ZeroInflatedBinomial` is failing

```
FAILED preliz/tests/test_mle.py::test_auto_recover[ZeroInflatedBinomial-params40] - OverflowError
```

**Expected behavior**
Test for `ZeroInflatedBinomial` should pass.

--- a/preliz/distributions/zi_binomial.py
+++ b/preliz/distributions/zi_binomial.py
@@ -180,6 +180,8 @@ def nb_ppf(q, psi, n, p, lower, upper):
 def nb_logpdf(psi, n, y, p):
     if y == 0:
         return np.log((1 - psi) + psi * (1 - p) ** n)
+    if y > n:
+        return 0
     else:
         return (
             np.log(psi)

